diff --git a/data/configs/current/R3.cfg b/data/configs/current/R3.cfg
index 048f4ac..62ab519 100644
--- a/data/configs/current/R3.cfg
+++ b/data/configs/current/R3.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 3969 bytes
+Current configuration : 3894 bytes
 !
-! Last configuration change at 19:21:15 UTC Wed Apr 15 2026
+! Last configuration change at 13:59:03 UTC Tue Apr 21 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -18,7 +18,6 @@ boot-start-marker
 boot-end-marker
 !
 !
-enable secret 9 $9$1eBcdxFH1tKLT.$vmmMe3Ga6dcIIS1TNXNsJpDau6cDMdZBqvLJU2z/JuY
 !
 no aaa new-model
 !
@@ -28,7 +27,7 @@ no aaa new-model
 !
 !
 !
-ip domain name lab.local
+ip domain name netconfig-ai
 !
 !
 !
@@ -98,14 +97,14 @@ crypto pki certificate chain SLA-TrustPoint
   D697DF7F 28
   	quit
 !
-license udi pid CSR1000V sn 9GMFKG73IQ1
+license udi pid CSR1000V sn 921V4WOEVAR
 diagnostic bootup level minimal
 memory free low-watermark processor 71464
 !
 !
 spanning-tree extend system-id
 !
-username admin privilege 15 secret 9 $9$3B0iqxDRyJz0Pk$m7BUyNhDadoQsuj2wK.QGlmBYGyrVZnnBnbxoNCYtu6
+username admin privilege 15 secret 9 $9$4S/U9GFDmiOxI.$8GeN7pb4N2cgV/fv6wpgD3vQpHQ4wWWYlxANe6kreCc
 !
 redundancy
 !