diff --git a/data/configs/current/R3.cfg b/data/configs/current/R3.cfg
index b00ef3b..5686d99 100644
--- a/data/configs/current/R3.cfg
+++ b/data/configs/current/R3.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 3977 bytes
+Current configuration : 4338 bytes
 !
-! Last configuration change at 15:23:20 UTC Tue Apr 21 2026
+! Last configuration change at 01:12:33 UTC Fri Jul 10 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -18,6 +18,7 @@ boot-start-marker
 boot-end-marker
 !
 !
+enable secret 9 $9$pFeR8tn9dNxuAk$eLVI/bg6yeg6pkO8cfp/ev2SY29AXnrSzgRJTYXsmDU
 !
 no aaa new-model
 !
@@ -27,7 +28,7 @@ no aaa new-model
 !
 !
 !
-ip domain name netconfig-ai
+ip domain name MNandGenAI
 !
 !
 !
@@ -97,14 +98,14 @@ crypto pki certificate chain SLA-TrustPoint
   D697DF7F 28
   	quit
 !
-license udi pid CSR1000V sn 921V4WOEVAR
+license udi pid CSR1000V sn 9OM8DBYB8XN
 diagnostic bootup level minimal
 memory free low-watermark processor 71464
 !
 !
 spanning-tree extend system-id
 !
-username admin privilege 15 secret 9 $9$4S/U9GFDmiOxI.$8GeN7pb4N2cgV/fv6wpgD3vQpHQ4wWWYlxANe6kreCc
+username admin privilege 15 secret 9 $9$ENGI5lA9euUl1k$R1OsBUc8YHuR./NHXrVhUrlqSPG97OBCGboO3pEVY6o
 !
 redundancy
 !
@@ -133,7 +134,7 @@ redundancy
 !
 !
 interface GigabitEthernet1
- ip address 172.16.184.32 255.255.255.0
+ ip address 172.16.151.234 255.255.255.0
  negotiation auto
  no mop enabled
  no mop sysid
@@ -159,11 +160,32 @@ interface GigabitEthernet4
  no mop enabled
  no mop sysid
 !
+interface GigabitEthernet5
+ no ip address
+ shutdown
+ negotiation auto
+ no mop enabled
+ no mop sysid
+!
+interface GigabitEthernet6
+ no ip address
+ shutdown
+ negotiation auto
+ no mop enabled
+ no mop sysid
+!
+interface GigabitEthernet7
+ no ip address
+ shutdown
+ negotiation auto
+ no mop enabled
+ no mop sysid
+!
 ip forward-protocol nd
 no ip http server
 ip http secure-server
 !
-ip route 0.0.0.0 0.0.0.0 172.16.184.1 name management_route
+ip route 0.0.0.0 0.0.0.0 172.16.151.1
 ip ssh version 2
 !
 !