diff --git a/data/configs/current/R8.cfg b/data/configs/current/R8.cfg
index da25a81..5e86c66 100644
--- a/data/configs/current/R8.cfg
+++ b/data/configs/current/R8.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4337 bytes
+Current configuration : 4327 bytes
 !
-! Last configuration change at 00:30:43 UTC Fri Jul 10 2026
+! Last configuration change at 01:09:30 UTC Fri Jul 10 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -141,7 +141,6 @@ interface GigabitEthernet1
 !
 interface GigabitEthernet2
  no ip address
- shutdown
  negotiation auto
  no mop enabled
  no mop sysid