diff --git a/data/configs/current/R5.cfg b/data/configs/current/R5.cfg
index 3bd8789..cfd5499 100644
--- a/data/configs/current/R5.cfg
+++ b/data/configs/current/R5.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4053 bytes
+Current configuration : 4098 bytes
 !
-! Last configuration change at 00:05:26 UTC Wed Jul 22 2026
+! Last configuration change at 02:17:24 UTC Fri Jul 24 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -100,6 +100,9 @@ crypto pki certificate chain SLA-TrustPoint
 !
 license udi pid CSR1000V sn 945RE05KYG1
 diagnostic bootup level minimal
+archive
+ path bootflash:/archive
+ maximum 14
 memory free low-watermark processor 71464
 !
 !