diff --git a/data/configs/current/R5.cfg b/data/configs/current/R5.cfg
index f6bc247..76c3d6b 100644
--- a/data/configs/current/R5.cfg
+++ b/data/configs/current/R5.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4031 bytes
+Current configuration : 4021 bytes
 !
-! Last configuration change at 00:29:44 UTC Fri Jul 10 2026
+! Last configuration change at 01:08:51 UTC Fri Jul 10 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -141,7 +141,6 @@ interface GigabitEthernet1
 !
 interface GigabitEthernet2
  no ip address
- shutdown
  negotiation auto
  no mop enabled
  no mop sysid