diff --git a/data/configs/current/R1.cfg b/data/configs/current/R1.cfg
index 4cc9894..49eeaf7 100644
--- a/data/configs/current/R1.cfg
+++ b/data/configs/current/R1.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4210 bytes
+Current configuration : 4283 bytes
 !
-! Last configuration change at 02:01:56 UTC Fri Jul 24 2026
+! Last configuration change at 02:08:13 UTC Fri Jul 24 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -100,7 +100,11 @@ crypto pki certificate chain SLA-TrustPoint
 !
 license udi pid CSR1000V sn 9S42I9WVYBG
 diagnostic bootup level minimal
+archive
+ path bootflash:/archive
+ maximum 14
 memory free low-watermark processor 71464
+file prompt quiet
 !
 !
 spanning-tree extend system-id
@@ -143,6 +147,7 @@ interface GigabitEthernet1
 interface GigabitEthernet2
  description local-facin-interface
  ip address 192.168.1.1 255.255.255.0
+ shutdown
  negotiation auto
  no mop enabled
  no mop sysid