diff --git a/data/configs/current/R1.cfg b/data/configs/current/R1.cfg
index 847394e..d923cde 100644
--- a/data/configs/current/R1.cfg
+++ b/data/configs/current/R1.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4002 bytes
+Current configuration : 4025 bytes
 !
-! Last configuration change at 23:47:24 UTC Tue Jul 21 2026
+! Last configuration change at 23:51:10 UTC Tue Jul 21 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -140,7 +140,7 @@ interface GigabitEthernet1
  no mop sysid
 !
 interface GigabitEthernet2
- no ip address
+ ip address 192.168.1.1 255.255.255.0
  negotiation auto
  no mop enabled
  no mop sysid