diff --git a/data/configs/current/R1.cfg b/data/configs/current/R1.cfg
index 3b34af9..430b38e 100644
--- a/data/configs/current/R1.cfg
+++ b/data/configs/current/R1.cfg
@@ -1,12 +1,11 @@
 Building configuration...
-Current configuration : 6180 bytes
+Current configuration : 6137 bytes
 !
-! Last configuration change at 15:19:51 UTC Tue Apr 21 2026
+! Last configuration change at 20:38:07 UTC Wed Apr 22 2026
 !
 version 17.3
 service timestamps debug datetime msec
 service timestamps log datetime msec
-! Call-home is enabled by Smart-Licensing.
 service call-home
 platform qfp utilization monitor load 80
 platform punt-keepalive disable-kernel-core