diff --git a/data/configs/current/R1.cfg b/data/configs/current/R1.cfg
index 430b38e..98ded36 100644
--- a/data/configs/current/R1.cfg
+++ b/data/configs/current/R1.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 6137 bytes
+Current configuration : 6149 bytes
 !
-! Last configuration change at 20:38:07 UTC Wed Apr 22 2026
+! Last configuration change at 04:32:17 UTC Thu Apr 23 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -173,8 +173,7 @@ interface GigabitEthernet1
  no mop sysid
 !
 interface GigabitEthernet2
- no ip address
- shutdown
+ ip address 10.0.1.1 255.255.255.252
  negotiation auto
  no mop enabled
  no mop sysid