diff --git a/data/configs/current/R1.cfg b/data/configs/current/R1.cfg
index 9bf0841..ec76ed6 100644
--- a/data/configs/current/R1.cfg
+++ b/data/configs/current/R1.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 3926 bytes
+Current configuration : 3945 bytes
 !
-! Last configuration change at 19:28:32 UTC Wed Apr 15 2026
+! Last configuration change at 19:29:56 UTC Wed Apr 15 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -140,6 +140,7 @@ interface GigabitEthernet1
  no mop sysid
 !
 interface GigabitEthernet2
+ description TO R2
  no ip address
  shutdown
  negotiation auto