diff --git a/data/configs/current/R1.cfg b/data/configs/current/R1.cfg
index f7bb3c3..4f4c3b0 100644
--- a/data/configs/current/R1.cfg
+++ b/data/configs/current/R1.cfg
@@ -1,5 +1,5 @@
 Building configuration...
-Current configuration : 4102 bytes
+Current configuration : 4112 bytes
 !
 ! Last configuration change at 00:39:22 UTC Wed Jul 22 2026
 !
@@ -134,7 +134,7 @@ redundancy
 !
 !
 interface GigabitEthernet1
- description external
+ description external connector
  ip address 172.16.151.233 255.255.255.0
  negotiation auto
  no mop enabled