diff --git a/data/configs/current/R1.cfg b/data/configs/current/R1.cfg
index 6a1e327..f7bb3c3 100644
--- a/data/configs/current/R1.cfg
+++ b/data/configs/current/R1.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4072 bytes
+Current configuration : 4102 bytes
 !
-! Last configuration change at 00:21:55 UTC Wed Jul 22 2026
+! Last configuration change at 00:39:22 UTC Wed Jul 22 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -142,18 +142,21 @@ interface GigabitEthernet1
 !
 interface GigabitEthernet2
  ip address 192.168.1.1 255.255.255.0
+ shutdown
  negotiation auto
  no mop enabled
  no mop sysid
 !
 interface GigabitEthernet3
  no ip address
+ shutdown
  negotiation auto
  no mop enabled
  no mop sysid
 !
 interface GigabitEthernet4
  no ip address
+ shutdown
  negotiation auto
  no mop enabled
  no mop sysid