diff --git a/data/configs/current/R1.cfg b/data/configs/current/R1.cfg
index 24a66fb..9bf0841 100644
--- a/data/configs/current/R1.cfg
+++ b/data/configs/current/R1.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 3891 bytes
+Current configuration : 3926 bytes
 !
-! Last configuration change at 19:21:30 UTC Wed Apr 15 2026
+! Last configuration change at 19:28:32 UTC Wed Apr 15 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -133,6 +133,7 @@ redundancy
 !
 !
 interface GigabitEthernet1
+ description TO EXTERNAL CONNECTOR
  ip address dhcp
  negotiation auto
  no mop enabled