diff --git a/data/configs/current/R1.cfg b/data/configs/current/R1.cfg
index 00756df..6a1e327 100644
--- a/data/configs/current/R1.cfg
+++ b/data/configs/current/R1.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4047 bytes
+Current configuration : 4072 bytes
 !
-! Last configuration change at 00:02:28 UTC Wed Jul 22 2026
+! Last configuration change at 00:21:55 UTC Wed Jul 22 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -166,6 +166,7 @@ ip route 0.0.0.0 0.0.0.0 172.16.151.1
 ip ssh version 2
 !
 !
+logging host 192.168.1.1
 !
 !
 !