diff --git a/data/configs/current/R1.cfg b/data/configs/current/R1.cfg
index 72fe9ec..ccf9f48 100644
--- a/data/configs/current/R1.cfg
+++ b/data/configs/current/R1.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4273 bytes
+Current configuration : 4255 bytes
 !
-! Last configuration change at 02:08:43 UTC Fri Jul 24 2026
+! Last configuration change at 18:44:00 UTC Sat Jul 25 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -104,7 +104,6 @@ archive
  path bootflash:/archive
  maximum 14
 memory free low-watermark processor 71464
-file prompt quiet
 !
 !
 spanning-tree extend system-id