diff --git a/data/configs/current/R1.cfg b/data/configs/current/R1.cfg
index 98ded36..96184e5 100644
--- a/data/configs/current/R1.cfg
+++ b/data/configs/current/R1.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 6149 bytes
+Current configuration : 6214 bytes
 !
-! Last configuration change at 04:32:17 UTC Thu Apr 23 2026
+! Last configuration change at 04:41:21 UTC Thu Apr 23 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -168,6 +168,7 @@ redundancy
 !
 interface GigabitEthernet1
  ip address 172.16.184.30 255.255.255.0
+ ip access-group 10 in
  negotiation auto
  no mop enabled
  no mop sysid
@@ -200,6 +201,8 @@ ip route 0.0.0.0 0.0.0.0 172.16.184.1 name management_route
 ip ssh version 2
 !
 !
+ip access-list standard 10
+ 10 permit any
 !
 !
 !