diff --git a/data/configs/current/R2.cfg b/data/configs/current/R2.cfg
index 4d63194..24a5e73 100644
--- a/data/configs/current/R2.cfg
+++ b/data/configs/current/R2.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4226 bytes
+Current configuration : 4206 bytes
 !
-! Last configuration change at 01:08:15 UTC Fri Jul 10 2026
+! Last configuration change at 05:26:18 UTC Fri Jul 17 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -147,14 +147,12 @@ interface GigabitEthernet2
 !
 interface GigabitEthernet3
  no ip address
- shutdown
  negotiation auto
  no mop enabled
  no mop sysid
 !
 interface GigabitEthernet4
  no ip address
- shutdown
  negotiation auto
  no mop enabled
  no mop sysid