diff --git a/data/configs/current/R2.cfg b/data/configs/current/R2.cfg
index eee6cbd..41cc965 100644
--- a/data/configs/current/R2.cfg
+++ b/data/configs/current/R2.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 3988 bytes
+Current configuration : 3894 bytes
 !
-! Last configuration change at 15:25:54 UTC Thu Apr 16 2026
+! Last configuration change at 14:00:21 UTC Tue Apr 21 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -18,7 +18,6 @@ boot-start-marker
 boot-end-marker
 !
 !
-enable secret 9 $9$MbVoXwhXDhyoIE$CPeyzKciK9ugjc88aiN/qMa5BZZjAbN4OxS6zRXoVvQ
 !
 no aaa new-model
 !
@@ -28,7 +27,7 @@ no aaa new-model
 !
 !
 !
-ip domain name lab.local
+ip domain name netconfig-ai
 !
 !
 !
@@ -98,14 +97,14 @@ crypto pki certificate chain SLA-TrustPoint
   D697DF7F 28
   	quit
 !
-license udi pid CSR1000V sn 9BCPRIB802D
+license udi pid CSR1000V sn 9PYODKX02ZS
 diagnostic bootup level minimal
 memory free low-watermark processor 71464
 !
 !
 spanning-tree extend system-id
 !
-username admin privilege 15 secret 9 $9$Sik3AHK.Z28hQU$AQpTIwdEFojeCriLGMYeaVfOFksf8fzddRZAGIDVAWQ
+username admin privilege 15 secret 9 $9$53NQZ7gIJniDEk$GwLsQDvD56zEncQ71BNbTpSRGwRG37bCTsenhdLPzwg
 !
 redundancy
 !
@@ -140,7 +139,6 @@ interface GigabitEthernet1
  no mop sysid
 !
 interface GigabitEthernet2
- description TO R1
  no ip address
  shutdown
  negotiation auto