diff --git a/data/configs/current/R2.cfg b/data/configs/current/R2.cfg
index fa8e4f6..5245ed0 100644
--- a/data/configs/current/R2.cfg
+++ b/data/configs/current/R2.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 3969 bytes
+Current configuration : 3988 bytes
 !
-! Last configuration change at 19:21:01 UTC Wed Apr 15 2026
+! Last configuration change at 15:18:39 UTC Thu Apr 16 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -140,6 +140,7 @@ interface GigabitEthernet1
  no mop sysid
 !
 interface GigabitEthernet2
+ description TO R1
  no ip address
  shutdown
  negotiation auto