diff --git a/data/configs/current/R2.cfg b/data/configs/current/R2.cfg
index 185f06f..4d63194 100644
--- a/data/configs/current/R2.cfg
+++ b/data/configs/current/R2.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4236 bytes
+Current configuration : 4226 bytes
 !
-! Last configuration change at 00:26:51 UTC Fri Jul 10 2026
+! Last configuration change at 01:08:15 UTC Fri Jul 10 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -141,7 +141,6 @@ interface GigabitEthernet1
 !
 interface GigabitEthernet2
  no ip address
- shutdown
  negotiation auto
  no mop enabled
  no mop sysid