diff --git a/data/configs/current/R2.cfg b/data/configs/current/R2.cfg
index 5245ed0..e2b04b5 100644
--- a/data/configs/current/R2.cfg
+++ b/data/configs/current/R2.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 3988 bytes
+Current configuration : 4007 bytes
 !
-! Last configuration change at 15:18:39 UTC Thu Apr 16 2026
+! Last configuration change at 15:24:09 UTC Thu Apr 16 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -155,6 +155,7 @@ interface GigabitEthernet3
  no mop sysid
 !
 interface GigabitEthernet4
+ description To R3
  no ip address
  shutdown
  negotiation auto