diff --git a/data/configs/current/R2.cfg b/data/configs/current/R2.cfg
index 52fa26d..d0754c9 100644
--- a/data/configs/current/R2.cfg
+++ b/data/configs/current/R2.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4236 bytes
+Current configuration : 4258 bytes
 !
-! Last configuration change at 05:31:11 UTC Fri Jul 17 2026
+! Last configuration change at 00:02:34 UTC Wed Jul 22 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -134,6 +134,7 @@ redundancy
 !
 !
 interface GigabitEthernet1
+ description external
  ip address 172.16.151.232 255.255.255.0
  negotiation auto
  no mop enabled