diff --git a/data/configs/current/R2.cfg b/data/configs/current/R2.cfg
index e7188d6..185f06f 100644
--- a/data/configs/current/R2.cfg
+++ b/data/configs/current/R2.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 3977 bytes
+Current configuration : 4236 bytes
 !
-! Last configuration change at 15:22:20 UTC Tue Apr 21 2026
+! Last configuration change at 00:26:51 UTC Fri Jul 10 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -18,6 +18,7 @@ boot-start-marker
 boot-end-marker
 !
 !
+enable secret 9 $9$zg53Y98ZxfcKPk$BsZK.Kjeg5y76ntWBpvam2HyYetamoGhaCf2ZWFDKpM
 !
 no aaa new-model
 !
@@ -27,7 +28,7 @@ no aaa new-model
 !
 !
 !
-ip domain name netconfig-ai
+ip domain name MNandGenAI
 !
 !
 !
@@ -97,14 +98,14 @@ crypto pki certificate chain SLA-TrustPoint
   D697DF7F 28
   	quit
 !
-license udi pid CSR1000V sn 9PYODKX02ZS
+license udi pid CSR1000V sn 95689PCZPWL
 diagnostic bootup level minimal
 memory free low-watermark processor 71464
 !
 !
 spanning-tree extend system-id
 !
-username admin privilege 15 secret 9 $9$53NQZ7gIJniDEk$GwLsQDvD56zEncQ71BNbTpSRGwRG37bCTsenhdLPzwg
+username admin privilege 15 secret 9 $9$DeA5MVGmk3lDDU$Zt25orWmXzkGg3E.FWdGgDik3pqQq5gtXxOuHZtITEM
 !
 redundancy
 !
@@ -133,7 +134,7 @@ redundancy
 !
 !
 interface GigabitEthernet1
- ip address 172.16.184.31 255.255.255.0
+ ip address 172.16.151.232 255.255.255.0
  negotiation auto
  no mop enabled
  no mop sysid
@@ -159,11 +160,25 @@ interface GigabitEthernet4
  no mop enabled
  no mop sysid
 !
+interface GigabitEthernet5
+ no ip address
+ shutdown
+ negotiation auto
+ no mop enabled
+ no mop sysid
+!
+interface GigabitEthernet6
+ no ip address
+ shutdown
+ negotiation auto
+ no mop enabled
+ no mop sysid
+!
 ip forward-protocol nd
 no ip http server
 ip http secure-server
 !
-ip route 0.0.0.0 0.0.0.0 172.16.184.1 name management_route
+ip route 0.0.0.0 0.0.0.0 172.16.151.1
 ip ssh version 2
 !
 !