diff --git a/data/configs/current/R4.cfg b/data/configs/current/R4.cfg
index 794129d..74ee93f 100644
--- a/data/configs/current/R4.cfg
+++ b/data/configs/current/R4.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4328 bytes
+Current configuration : 4338 bytes
 !
-! Last configuration change at 01:08:32 UTC Fri Jul 10 2026
+! Last configuration change at 01:12:38 UTC Fri Jul 10 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -141,6 +141,7 @@ interface GigabitEthernet1
 !
 interface GigabitEthernet2
  no ip address
+ shutdown
  negotiation auto
  no mop enabled
  no mop sysid