diff --git a/data/configs/current/R7.cfg b/data/configs/current/R7.cfg
index 9ba07d9..5aeac5b 100644
--- a/data/configs/current/R7.cfg
+++ b/data/configs/current/R7.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4359 bytes
+Current configuration : 4380 bytes
 !
-! Last configuration change at 00:05:16 UTC Wed Jul 22 2026
+! Last configuration change at 01:34:59 UTC Fri Jul 24 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -188,6 +188,7 @@ ip http secure-server
 !
 ip route 0.0.0.0 0.0.0.0 172.16.190.1
 ip ssh version 2
+ip scp server enable
 !
 !
 !