diff --git a/data/configs/current/R7.cfg b/data/configs/current/R7.cfg
index 5aeac5b..84f50bb 100644
--- a/data/configs/current/R7.cfg
+++ b/data/configs/current/R7.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4380 bytes
+Current configuration : 4425 bytes
 !
-! Last configuration change at 01:34:59 UTC Fri Jul 24 2026
+! Last configuration change at 02:17:46 UTC Fri Jul 24 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -100,6 +100,9 @@ crypto pki certificate chain SLA-TrustPoint
 !
 license udi pid CSR1000V sn 9LFDAUTV0AN
 diagnostic bootup level minimal
+archive
+ path bootflash:/archive
+ maximum 14
 memory free low-watermark processor 71464
 !
 !