diff --git a/data/configs/current/R6.cfg b/data/configs/current/R6.cfg
index 5ed64c3..b73d3f2 100644
--- a/data/configs/current/R6.cfg
+++ b/data/configs/current/R6.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4272 bytes
+Current configuration : 4293 bytes
 !
-! Last configuration change at 23:15:30 UTC Wed Jul 22 2026
+! Last configuration change at 01:34:49 UTC Fri Jul 24 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -181,6 +181,7 @@ ip http secure-server
 !
 ip route 0.0.0.0 0.0.0.0 172.16.190.1
 ip ssh version 2
+ip scp server enable
 !
 !
 !