diff --git a/data/configs/current/R6.cfg b/data/configs/current/R6.cfg
index b0d592a..3226bfe 100644
--- a/data/configs/current/R6.cfg
+++ b/data/configs/current/R6.cfg
@@ -1,5 +1,5 @@
 Building configuration...
-Current configuration : 4257 bytes
+Current configuration : 4272 bytes
 !
 ! Last configuration change at 00:05:22 UTC Wed Jul 22 2026
 !
@@ -134,7 +134,7 @@ redundancy
 !
 !
 interface GigabitEthernet1
- description external
+ description Link-to-Management-Core
  ip address 172.16.190.27 255.255.255.0
  negotiation auto
  no mop enabled