diff --git a/data/configs/current/R6.cfg b/data/configs/current/R6.cfg
index b73d3f2..f272c5b 100644
--- a/data/configs/current/R6.cfg
+++ b/data/configs/current/R6.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4293 bytes
+Current configuration : 4338 bytes
 !
-! Last configuration change at 01:34:49 UTC Fri Jul 24 2026
+! Last configuration change at 02:17:35 UTC Fri Jul 24 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -100,6 +100,9 @@ crypto pki certificate chain SLA-TrustPoint
 !
 license udi pid CSR1000V sn 9GX66ZBD079
 diagnostic bootup level minimal
+archive
+ path bootflash:/archive
+ maximum 14
 memory free low-watermark processor 71464
 !
 !