diff --git a/data/configs/current/R6.cfg b/data/configs/current/R6.cfg
index e8f8447..faf2d09 100644
--- a/data/configs/current/R6.cfg
+++ b/data/configs/current/R6.cfg
@@ -1,7 +1,7 @@
 Building configuration...
-Current configuration : 4235 bytes
+Current configuration : 4225 bytes
 !
-! Last configuration change at 00:30:08 UTC Fri Jul 10 2026
+! Last configuration change at 01:09:03 UTC Fri Jul 10 2026
 !
 version 17.3
 service timestamps debug datetime msec
@@ -141,7 +141,6 @@ interface GigabitEthernet1
 !
 interface GigabitEthernet2
  no ip address
- shutdown
  negotiation auto
  no mop enabled
  no mop sysid